tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -129,4 +129,25 @@ Feature: create auth-app token
         "minItems": 0,
         "maxItems": 0
       }
+      """
+
+
+  Scenario: try to create auth-app token without expiry
+    When user "Alice" tries to create app token with expiration time "" using the auth-app API
+    Then the HTTP status code should be "400"
+    And the content in the response should match the following content:
+      """
+      error parsing expiry. Use e.g. 30m or 72h
+
+      """
```

Comment:
```suggestion
    And the content in the response should match the following content:
      """
      error parsing expiry. Use e.g. 30m or 72h
      """
```

Reply:
can't we do something for this

Reply:
you can use a step that checks the string using contains assertion